### PR TITLE
Update readme for v3 settings

### PR DESCRIPTION
**Pull request by dshire**
Updated the embedding.md readme file that is used to config the embedded webchat with the right settings. It now references v3 settings properly. also updated the example codesanbox, cleaned up some stuff and added a full interface for reference.

### CODE REVIEW

**Comment by sushmi21** [comment]
We should remove the codesandbox example in line 10 and 11

**Comment by sushmi21** [comment]
> We should remove the codesandbox example in line 10 and 11, right? It still shows v2 example in codesandbox

I see that only the endpoint settings example has v3 embedded in codesandbox. Shouldn't we replace all v2 codesandbox examples with v3?

File: docs/embedding.md [hunk 1]
```diff
@@ -71,131 +56,108 @@ You can pass [Webchat Options](#webchat-options) as an additional argument to th
...
+| - | - | - | - |
+| title | string | `""` | The text that will be shown as header title of the Webchat |
+| logoUrl | string | `""` | The Logo shown in the header of the Webchat |
+| useOtherAgentLogo | boolean | `false` | Enables setting specific names and logos for bot and agent avatars. By default title and logoUrl will be used for the avatars. |
+| botAvatarName | string | `""` | Name for the AI Agent avatar which will be displayed above each AI Agent message. |
+| botLogoUrl | string | `""` | AI Agent avatar which will be displayed above each AI Agent message. Recommended img size: 28px x 28px |
+| agentAvatarName | string | `""` | Name for the Human Agent avatar which will be displayed above each Human Agent message |
+| agentLogoUrl | string | `""` | Human Agent avatar which will be displayed above each Human Agent message. Recommended img size: 28px x 28px |
+| inputAutogrowMaxRows | number | `4` | Maximum Number of Input Rows  |
+| enableInputCollation | boolean | `true` | If enabled, messages will be combined into a single message, dependent on the time set with `inputCollationTimeout` |
```
Comment (sushmi21) [inline docs/embedding.md:101]: default Value should be false. See [spec ](https://cognigy.visualstudio.com/Engineering/_git/technical-design-documents/pullrequest/24778?_a=files&path=/docs/Products/Webchat/concepts/webchat-v3-endpoint/index.md). I have fixed this in my [tests PR](https://github.com/Cognigy/WebchatWidget/pull/380/files#diff-278c6a88ca5523f8d2b36db48728f8a3c4129c3ef2af21d21d31d5492b9b8a28)

File: docs/embedding.md [hunk 2]
```diff
@@ -205,10 +167,140 @@ All business hours settings require the setting `awaitEndpointConfig` as prerequ
...
+| - | - | - | - |
+| text | string | `""` | Message displayed above Webchat icon for user engagement |
+| showInChat | boolean | `false` | Enable to also show the Teaser Message text in the chat window |
+| conversationStarters | object | [Conversation Starters](#conversation-starters) | Configure Conversation Starters, each will be an action button below the Teaser Message |
+
+#### Chat Options
+| Name | Type | Default | Description |
+| - | - | - | - |
+| enabled | boolean | `false` | Enable Chat Options Feature |
+| title | string |  `"Chat Options"` | Chat Options header title |
```
Comment (sushmi21) [inline docs/embedding.md:227]:
```suggestion
| title | string |  `"Chat options"` | Chat Options header title |
```

**Review by sushmi21 — CHANGES_REQUESTED** [review]

**Comment by dshire** [comment]
> > We should remove the codesandbox example in line 10 and 11, right? It still shows v2 example in codesandbox
> 
> I see that only the endpoint settings example has v3 embedded in codesandbox. Shouldn't we replace all v2 codesandbox examples with v3?

No I only exchanged the settings one as that one was technically wrong. the others are still correct. once v3 will be latest official release, we need to change all of them anyway, as the v3 ones currently have a beta url

File: docs/embedding.md (hunk 1 above)
Comment (dshire) [reply docs/embedding.md:101]: fixed

File: docs/embedding.md [hunk 3]
```diff
@@ -71,131 +56,108 @@ You can pass [Webchat Options](#webchat-options) as an additional argument to th
...
+| botMessageColor | string | `""` | AI Agent messages background color |
+| userMessageColor | string | `""` | User messages background color |
+| textLinkColor | string | `""` | Text link color |
+
+#### Behavior
+| Name | Type | Default | Description |
+| - | - | - | - |
+| enableTypingIndicator | boolean | `true` | Enables a typing indicator while the AI Agent is replying |
+| messageDelay | number | `500` | Wait time between AI Agent messages in miliseconds |
+| inputPlaceholder | string | `"Type something…"` | Placeholder for the input field |
```
[the remark this answers is not in the document]
Comment (dshire) [reply docs/embedding.md:126]: fixed

File: docs/embedding.md [hunk 4]
```diff
@@ -205,10 +167,140 @@ All business hours settings require the setting `awaitEndpointConfig` as prerequ
...
+| enabled | boolean | `true` | Enable quick replies in the Chat Options |
+| sectionTitle | string | `"People are also interested in"` | Configure the title for the Quick Reply Section |
+| quickReplies | array of [Action Buttons](#action-button) | `[]` | Configure action buttons to show as quick replies |
+
+##### Chat Options Rating Section
+| Name | Type | Default | Description |
+| - | - | - | - |
+| enabled | string | `"once"` | Allowed values: `"no" \| "once" \| "always"`. Decide how often a user can give feedback from this section. Hidden not enabled. |
+| title | string | `"Please rate your chat experience"` | Configure the title for the Rating Section |
+| commentPlaceholder | string | `"Type something here"` | Placeholder text that will be displayed in the rating comment field |
```
[the remark this answers is not in the document]
Comment (dshire) [reply docs/embedding.md:247]: fixed

File: docs/embedding.md [hunk 5]
```diff
@@ -71,131 +56,108 @@ You can pass [Webchat Options](#webchat-options) as an additional argument to th
...
+| enableGenericHTMLStyling | boolean | `false` | If this is active, additional generic styling will be applied to HTML content inside regular text messages. |
+| disableHtmlContentSanitization | boolean | `false` | By default, potentially malicious HTML content like 'onclick' or 'onload' attributes are removed before rendering.  |
+| disableUrlButtonSanitization | boolean | `false` | By default, 'JavaScript URLs' starting with javascript: will get removed. |
+| watermark | string | `"default"` | Allowed values: `"default" \| "custom" \| "none"` |
+| watermarkText | string | `"Powered by Cognigy.AI"` | This will be used if watermark is set to custom |
+
+#### Colors
+| Name | Type | Default | Description |
+| - | - | - | - |
+| primaryColor | string | `""` | Primary color |
```
[the remark this answers is not in the document]
Comment (dshire) [reply docs/embedding.md:114]: fixed

File: docs/embedding.md (hunk 2 above)
Comment (dshire) [reply docs/embedding.md:227]: fixed

**Pull request merged**